include/onnxruntime/core/graph/basic_types.h

```diff
@@ -27,8 +27,6 @@ using ProviderType = const std::string&;
 // instead of std::unordered_map<std::string, foo, [std::less<foo>]>.
 
 using NodeAttributes = std::unordered_map<std::string, ONNX_NAMESPACE::AttributeProto>;
-class IOnnxRuntimeOpSchemaCollection;
```

Comment:
Ryan proposed this - the ...Ptr type looks confusing since it might by Type*, unique_ptr<Type> or shared_ptr<Type>, in some recent changes such kind of definitions were being removed.